src/content/docs/errors-inbox/group-request-errors.mdx

```diff
@@ -0,0 +1,222 @@
+
+* **Investigate individual request errors**: Select a request error report to view its distributed trace, event trail, response body, attributes, and other relevant details necessary for effective debugging.
+
+* **Update error status**: Mark the request error as **Resolved** or assign it to a specific team member who can own and examine the issue, and then deploy a fix.
+
+## Request error details [#error-details]
+
+On the request errors page, click into a specific error to see:
+* **User journeys**: Displays the different paths and actions a user performed that led to the error.
+* **All occurrences chart**: Displays the frequency of a request error over the selected period, starting from its initial detection. You can view either the aggregated data of all occurrences or a breakdown by app version.
+* **Error type breakdown**: Displays the distribution of this request errors across different operating system versions or affected devices.
+
+Resolved errors include a banner containing details on the user who resolved the request error and the resolution timestamp. Note that mobile monitoring's data retention policies apply, allowing you to filter by resolved errors for historical analysis when needed.
+
+<img
+    title="Request errors details page"
+    alt="Screenshot of the request errors details page"
+    src="/images/errors-inbox_screenshot-crop_request-errors-details-page.webp"
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT>**: Select a request error to open the request errors details page.
+</figcaption>
+
+### Profiles [#error-profiles]
+
+Error profiles provide visual details about significant differences in the frequency of different values for HTTP error events. For each attribute, the error profile includes:
+* A heatmap showing how the error's attribute is distributed for values that deviate the most
+* A label comparing the error attribute's distribution to that of other errors
+
+<img
+    title="Request errors profiles"
+    style={{align: "left" }}
+    width="50%"
+    alt="Screenshot of the request errors profiles section"
+    src="/images/errors-inbox_screenshot-crop_request-errors-profiles.webp"
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT>**: Select a request error to open the request errors details page and view the heatmap distribution of the attributes.
+</figcaption>
+
+### Triage [#triage-errors]
+
+The triage section associates the specific error occurrence you are viewing with its [system-created error group](/docs/errors-inbox/errors-inbox/#groups). These system-created error groups are identified by a unique fingerprint. It is this unique fingerprint that allows you to triage error groups by status updates or assignments.
```

Comment:
@ally-sassman where is this in the UI?

Reply:
These are all subsections of the request errors details page, so you'd click into an error or error group and see all of these sections on the popover (triage, profiles, heatmap, etc.)